BraveWallet/Extensions/WalletNumberFormatterExtensions.swift

```diff
@@ -7,8 +7,10 @@ import Foundation
 
 extension NumberFormatter {
   /// Currency number formatter in USD
-  static let usdCurrencyFormatter = NumberFormatter().then {
-    $0.numberStyle = .currency
-    $0.currencyCode = CurrencyCode.usd.code
+  static func usdCurrencyFormatter() -> NumberFormatter {
+    return NumberFormatter().then {
+      $0.numberStyle = .currency
+      $0.currencyCode = CurrencyCode.usd.code
+    }
```

Comment:
Why is this now a function? Is it just so it creates a new formatter each time its accessed? If so then since there's no args this can just be a `static var`

Reply:
There are couple places need to change the minimumFractionDigits and maximumFractionDigits, since Solana transaction has very little transaction fee. we need to display enough decimals for the fiat value. However, the rest we will keep 2 decimals. 
It will also break unit tests since tests are running parallelly

Reply:
Right so its just to get a new number formatter each time so that you can mutate it? If so, a computed `static var` should be fine.  If you are going to keep it as a static func then it should have args of some sort (maybe what you plan on changing?)

Reply:
ah smart boy ... why not `static var` to fix `static let` 😂

Reply:
fixed in [1aaf290](https://github.com/brave/brave-ios/pull/5746/commits/1aaf290ebb80a10973b88d03bca9c0ed1647cf34)